File: examples/rpc/zmq/request-reply/clientToWhovianHost.py
import zmq
socketContext = zmq.Context()
socket = socketContext.socket(zmq.REQ)
socket.connect("tcp://whovian:%s" % '5556')

for i in range(5):
   socket.send_string("hello from python %d" % i)
   print(socket.recv_string())

File: examples/microservices/trenaGeneModel/clientToWhovianHost.R
library(rzmq)
library(jsonlite)
context = init.context()
socket = init.socket(context,"ZMQ_REQ")
connect.socket(socket,"tcp://whovian:5556")

i <- 0
while (i < 5) {
   send.raw.string(socket, sprintf("hello from R: %d", i))
   s <- receive.string(socket)
   print(s)
   i <- i + 1
   }
